libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/useDrillTargetTypeItems.ts

```diff
@@ -47,6 +48,9 @@ export const useDrillTargetTypeItems = (): IDrillTargetType[] => {
             id: DRILL_TARGET_TYPE.DRILL_DOWN,
             title: intl.formatMessage(messages.drillDownConfig),
             disabled: true,
+            disableTooltipMessage: disableDrillDown
+                ? intl.formatMessage(messages.disableDrillDownToolTip)
+                : "",
```

Comment:
Undefined would be better than an empty string.

Reply:
I have updated this. Thank you!